Build.PL

```diff
@@ -1,80 +1,81 @@
-use 5.010_001;

```

Comment:
Looks like you changed the entire file from CRLF to LF. While this might be a welcome change by itself, it's not a part of the PR.